nttt/utilities.py

```diff
@@ -55,4 +57,54 @@ def find_files(src, file_names=[], extensions=[]):
             if valid_file:
                 files_found.append(fpath)
 
-    return files_found
\ No newline at end of file
+    return files_found
+
+
+#PUB-9 utilities. These methods are for trimming spaces on specific markdown like ** or _. The main method is trim_spaces_on_specific_markdown which uses trim_spaces_on_specific_markdown_breakdown and strings_with_specific_markdown_and_spaces.
+ 
+
+regular_expressions_matrix_for_trimming_spaces = [
+    {'character':'_', 'regular_expression_for_wrong_left':r"_[ ]+[a-z]+[ ]*_", 'regular_expression_for_wrong_right':r"_[ ]*[a-z]+[ ]+_", "regular_expression_for_correct":r"_[a-z]*_"},
```

Comment:
I think a-z is way too restrictive. It probably should also include capital letters, numbers and underscore (valid character in identifiers).

The original request from Nina also mentions that the problem should be fixed for phrases that contain spaces. On the other hand adding space character in the middle of the regex may lead to some nasty false positives. Consider for example:

> ** is another math operator operator in Python. For example 5 ** 3 = 125

And getting those false positives is even worse than not having the check at all because it may spoil perfect translations.

So I'd suggest to re-evaluate this requirement with Nina and Maja highlighting all possible risks.

Reply:
Another option would be to introduce feature flags (command line arguments) that enable/disable those "dangerous" features.